indico/modules/events/sessions/templates/management/session_person_list.html

```diff
@@ -0,0 +1,85 @@
+<div id="persons-list" class="i-box titled">
+    <div class"i-box-title">
```

Comment:
missing `=` - but do we really want an ibox around it? to me it looks better without one